custom_components/wundasmart/translations/en.json

```
@@ -27,7 +27,11 @@
               "scan_interval": "Poll Interval",
               "connect_timeout": "Connect Timeout",
               "read_timeout": "Read Timeout",
-              "ping_interval": "Heartbeat Interval"
+              "ping_interval": "Heartbeat Interval",
+              "separate_room_devices": "Separate Room Devices"
+            },
+            "data_description": {
+              "separate_room_devices": "When enabled, each room becomes a separate device. When disabled, all entities are grouped under the hub device (legacy mode). WARNING: Switching from enabled to disabled may leave orphaned room devices that need to be manually removed from Settings > Devices & Services > Devices."
```

```suggestion
              "separate_room_devices": "When enabled, separate devices are created for each room. By default, all entities are grouped under a single device."
```

Reworded and removed warning since it seems that the cleanup should now be done automatically.